exe/cdc_agent/config.ini

```diff
@@ -0,0 +1,3 @@
+[mqtt]
```

Comment:
Can we get rid of this file and put these configuration elements into config.json?